modules/account-lib/test/unit/coin/eth/transactionBuilder/contractCall.ts

```diff
@@ -54,4 +54,23 @@ describe('Eth contract call transaction builder', () => {
     txBuilder.contract(contractAddress);
     await txBuilder.build().should.be.rejectedWith('Invalid transaction: missing contract call data field');
   });
+
+  it('should properly build tx for eth2 staking staking deposit', async () => {
+    // Eth2 staking contract
+    txBuilder.contract('0x00000000219ab540356cbb839cbe05303d7705fa');
+    // Sample data from https://etherscan.io/tx/0x4c5c36a2a41843ca0f304f4e5d49f4d97cd7b2654305866ccd747c4d44ea9827
+    const data =
```

Comment:
Please move into fixture instead of defining test data inline

Reply:
moved into resources file, thanks for the suggestion